bundles/io/org.eclipse.smarthome.io.transport.mqtt/src/main/java/org/eclipse/smarthome/io/transport/mqtt/MqttBrokerConnection.java

```diff
@@ -151,17 +156,16 @@ public void messageArrived(String topic, MqttMessage message) {
     @NonNullByDefault({})
     protected static class ConnectionCallbacks implements IMqttActionListener {
         private final MqttBrokerConnection c;
+        private final Runnable cancelTimeoutFuture;
 
         public ConnectionCallbacks(MqttBrokerConnection c) {
             this.c = c;
+            this.cancelTimeoutFuture = c::cancelTimeoutFuture;
```

Comment:
How do you call this java feature?

Reply:
[Method References] Similar to function pointers in C/C++ or lambda expressions.

[Method References]: https://docs.oracle.com/javase/tutorial/java/javaOO/methodreferences.html